docs/datasets.md

```diff
@@ -333,7 +333,8 @@ VALIDATION |     19,867
 High-quality version of the CELEBA
 dataset, consisting of 30000 images in 1024 x 1024 resolution.
 
-WARNING: this dataset currently requires you to prepare images on your own.
+WARNING: This dataset currently requires you to prepare images on your own. Tensorflow also provides an image generator 
```

Comment:
The datasets.md file is generated automatically using scripts/document_datasets.py and information from the datasets.

Please change the dataset descriptions in corresponding files instead.

Reply:
Thanks for the feedback, I've made the changes in #130